### PR TITLE
adding BA + PEs for Desikan-Killiany Atlas

### PR DESCRIPTION
**Pull request by lzehl**
the BA only contains the basic information (needs to be extended over time)

### CODE REVIEW

**Comment by lzehl** [comment]
@UlrikeS91 I've added 3 missing PEs, but it is complete now. Ready for Review :)

File: instances/atlas/brainAtlas/Desikan-KillianyAtlas.jsonld [hunk 1]
```diff
@@ -0,0 +1,165 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/brainAtlas/AAL1",
```
Comment (UlrikeS91) [inline instances/atlas/brainAtlas/Desikan-KillianyAtlas.jsonld]: This is wrong, isn't it? Should be something like ".../brainAtlas/DKA", right?

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
Just the at_id for the atlas is wrong. 

Everything else looks good. Both content and jsonld format look correct. The latter I know for sure, the content just seems logical based on my anatomical knowledge (parents seem like correct choices, UBERON terms seem to be the same or similar, etc.).

File: instances/atlas/brainAtlas/Desikan-KillianyAtlas.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/atlas/brainAtlas/Desikan-KillianyAtlas.jsonld]: yes.... copy/paste error.... it is corrected now

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**